ArduPlane/ArduPlane.cpp

```diff
@@ -76,6 +76,7 @@ const AP_Scheduler::Task Plane::scheduler_tasks[] = {
     SCHED_TASK(Log_Write_Fast,         25,    300),
     SCHED_TASK(update_logging1,        10,    300),
     SCHED_TASK(update_logging2,        10,    300),
+    SCHED_TASK(update_soaring,         50,    400),
```

Comment:
The 50Hz is intended, but I removed a bunch of code from SoaringController that used rate_hz.  That code wasn't being used or properly tested so far, and was relying on an incorrect value for rate_hz.